database/rrdset.c

```diff
@@ -388,6 +388,7 @@ void rrdset_free(RRDSET *st) {
         case RRD_MEMORY_MODE_ALLOC:
         case RRD_MEMORY_MODE_NONE:
         case RRD_MEMORY_MODE_DBENGINE:
+            freez(st->chart_uuid);
```

Comment:
I can only see an allocation site for DBENGINE, why is this freed for ALLOC and NONE?

Reply:
That's wrong I will fix it